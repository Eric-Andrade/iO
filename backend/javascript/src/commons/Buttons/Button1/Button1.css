.button1{
    border-radius: 3px;
    padding: 0.25em 1em;
    margin: 0 1em;
    background: #1e1e1e;
    border: 2px solid #f98484;
}
