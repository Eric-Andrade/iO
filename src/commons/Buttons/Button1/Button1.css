.button1{
    border-radius: 3px;
    padding: 0.25em 1em;
    margin: 0 1em;
    background: #ffffff;
    color: #db7093;
    border: 2px solid #db7093;
}