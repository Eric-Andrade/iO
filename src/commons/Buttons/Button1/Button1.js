import React from 'react'
import styles from './Button1.css'
// import styled, { css } from 'styled-components'

// const Button = styled.button`
//   border-radius: 3px;
//   padding: 0.25em 1em;
//   margin: 0 1em;
//   background: ${props => props.theme.WHITE};
//   color: ${props => props.theme.PRIMARY};
//   border: 2px solid ${props => props.theme.PRIMARY};

//   ${props => props.primary && css`
//     background: ${props => props.theme.PRIMARY};
//     color: white;
//   `}
// `

const SimpleButton = ({ children, onPress, disabled }) => (
  <button className={styles.button1} onClick={onPress} disabled={disabled} >
    {children}
  </button>
  )

export default SimpleButton
