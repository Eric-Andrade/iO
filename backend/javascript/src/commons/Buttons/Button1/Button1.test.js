import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Button1 from './Button1'

test('Button1 component should render as expected', () => {
    const component = shallow(<Button1 />)
    const  tree = toJson(component)
    expect(tree).toMatchSnapshot()
})
