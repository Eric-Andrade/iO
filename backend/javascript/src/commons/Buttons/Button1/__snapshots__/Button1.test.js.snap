// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button1 component should render as expected 1`] = `
<button
  className="button1"
/>
`;
